hw/ip/prim/lint/prim_clock_div.waiver

```diff
@@ -4,8 +4,15 @@
 #
 # waiver file for prim_clock_div
 
+waive -rules CLOCK_EDGE -location {prim_clock_div.sv} -msg {Falling edge of clock 'clk_i' used here, should use rising edge} \
+      -comment "The clock switch signal is synchronized on negative edge to ensure it is away from any transition"
+
 waive -rules DUAL_EDGE_CLOCK -location {prim_clock_div.sv} -regexp {.*} \
       -comment "The clock switch signal is synchronized on negative edge to ensure it is away from any transition"
 
 waive -rules CLOCK_MUX -location {prim_clock_div.sv} -regexp {.*reaches a multiplexer here, used as a clock.*} \
       -comment "A mux is used during scan bypass, and for switching between div by 2 and div by 1 clocks"
+
+waive -rules CLOCK_USE -location {rv_dm.sv} \
+      -msg {'clk_i' is connected to 'prim_clock_mux2' port 'clk1_i',} \
+      -comment "This clock usage is OK and is due to hand instantiated primitives on the clock line."
```

Comment:
This looks like being in the wrong waiver file.

Reply:
oh yeah, thanks for catching.